plasmapy/particles/particle_class.py

```diff
@@ -2023,7 +2023,119 @@ def mass(self, m: u.kg):
                 ) from exc
 
 
-# TODO: Describe valid particle representations in docstring of particle_like
+# TODO: Need to incorporate particle_like into @particle_input
+
+# TODO: Decide if DimensionlessParticle should be in particle_like
+
+particle_like = Union[str, Integral, Particle, CustomParticle]
+
+particle_like.__doc__ = """
+An `object` is particle-like if it can uniquely identify an ion, atom,
+isotope, special particle, or custom particle.
```

Comment:
```suggestion
An `object` is particle-like if it can be identified as an instance of 
`~plasmapy.particles.particle_class.Particle` or 
`~plasmapy.particles.particle_class.CustumParticle`, or cast into one.
```